lms/templates/homework.html

```diff
@@ -9,7 +9,7 @@
 
     <section>
       <input type="hidden" name="problem_id" value="${ problem['name'] }">
-      <input type="submit" value="Check">
+      <input type="submit" value="Submit">
```

Comment:
Don't see any code usages of it, and if you look at the history, it's not been touched since January 2012. Delete it!